src/renderer/components/opds/FeedList.tsx

```diff
@@ -44,9 +44,8 @@ export class FeedList extends React.Component<OpdsListProps, null> {
                 <ul>
                     { this.props.feeds.map((item, index) => {
                         return (
-                            <li>
+                            <li key={item.title}>
```

Comment:
Using `item.title` is not a valid solution, as there is no guarantee of unicity. See screenshot:

![ThoriumOPDSFeedUniqueKeys](https://user-images.githubusercontent.com/625201/61552820-beff0380-aa50-11e9-9ff2-ba573348d63b.png)

Reply:
Suggestion:
```
key={`feed-${index}`}
```

Reply:
Or:
```
key={item.identifier}
```